src/Compilers/CSharp/Portable/BoundTree/BoundExpressionExtensions.cs

```diff
@@ -42,6 +44,45 @@ public static RefKind GetRefKind(this BoundExpression node)
                 case BoundKind.ImplicitIndexerAccess:
                     return ((BoundImplicitIndexerAccess)node).IndexerOrSliceAccess.GetRefKind();
 
+                case BoundKind.ConditionalOperator:
+                    {
+                        var conditionalOperator = (BoundConditionalOperator)node;
+                        if (!conditionalOperator.IsRef)
+                        {
+                            return RefKind.None;
+                        }
+                        {
```

Comment:
>{ [](http://example.com/codeflow?start=24&length=1)

The extra blocks looks strange, I do not think they help readability of the code.